perf/time_series_updater.go

```diff
@@ -59,12 +60,13 @@ func NewPerformanceAnalysisService(baseURL, user string, token string) Performan
 // ReportUpdatedTimeSeries takes a TimeSeriesModel and tries to report its data to a PerformanceAnalysisService.
 func (spc *performanceAnalysisAndTriageClient) ReportUpdatedTimeSeries(ctx context.Context, series TimeSeriesModel) error {
 	startAt := time.Now()
+	series.GUID = utility.RandomString()
 
 	if err := spc.doRequest(http.MethodPost, spc.baseURL+"/time_series/update", ctx, series); err != nil {
 		return errors.WithStack(err)
 	}
 
-	grip.Debug(message.Fields{
+	grip.Info(message.Fields{
```

Comment:
Log `series.GUID` in this message. That way we can correlate the cedar and SPS logs.

Reply:
It's logged with `"update": series,` line since it is part of the `TimeSeriesUpdate` struct.

Reply:
separately, do we also want to log the error case (atm we're [returning on error](https://github.com/evergreen-ci/cedar/blob/7141126a0841e2a49ba7dde72b1dbfa101679bfd/perf/time_series_updater.go#L64) before we hit this)? For example, if we get `resp.StatusCode != 200` from SPS?

Reply:
Oh, I missed that, thanks. LGTM, but not resolving this thread since it has @ybrill 's question in it.

Reply:
That will get logged [here](https://github.com/evergreen-ci/cedar/blob/7141126a0841e2a49ba7dde72b1dbfa101679bfd/units/time_series_update.go#L130-L134).